tools/check-rule-examples.js

```diff
@@ -7,10 +7,13 @@
 const { parse } = require("espree");
 const { readFile } = require("fs").promises;
 const glob = require("glob");
+const matter = require("gray-matter");
 const markdownIt = require("markdown-it");
 const markdownItContainer = require("markdown-it-container");
 const { promisify } = require("util");
+const meta = require("../docs/src/_data/rules_meta.json");
```

Comment:
This file is autogenerated during releases and generally doesn't need to be updated in PRs that add or remove rules (for example, https://github.com/eslint/eslint/pull/17625). Perhaps we could use `require("../lib/rules")` for this purpose?